tools/lkl/tests/test.sh

```diff
@@ -124,6 +124,10 @@ lkl_test_exec()
     fi
 
     $SUDO $WRAPPER $file "$@"
+
```

Comment:
Should we run this when we collect the results with find ... in the circle.yml setup?

Reply:
agree. I'll fix it.

Reply:
this is also moved to .circleci/config.yml.